docs/overview/web3auth-for-wallets.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Web3Auth for Wallets
+image: "/docs/images/docs-meta-cards/documentation-card.png"
+displayed_sidebar: docs
+description: "Web3Auth for Wallets | Documentation - Web3Auth"
+---
+
+import Image1 from "@site/static/contents/web3auth-architecture.png";
+
+---
+
+One of the most important use cases of Web3Auth is to integrate it within wallets. This is because Web3Auth itself is not a wallet, it is wallet
+infrastructure that can be plugged into your own wallet. Web3Auth results in a standard cryptographic key provider specific to the user and
+application. This key provider can be used to sign transactions, messages, or making any other wallet transaction.
+
+## Using Web3Auth to build your Wallet
+
+---
+
+A wallet is basically a wrapper around the user's private key, helping them manage their account on a much better level. With Web3Auth, you can make
+the UI flows according to your own needs and requirements, while not thinking about the key generation and management part. Web3Auth will take care of
+that for you. You can even save additional private keys safely within your application with Web3Auth.
+
+Using Web3Auth for your own wallet flows has the following advantages:
+
+1. The wallet owns the whole experience, with minimal external branding. You can even surpass that (have **zero Web3Auth Branding**) if you use
+   [Web3Auth Self Host](/self-hosting).
+2. You're even able to use your own OAuth 2.0 / Single Sign-On or any other flow for validating your user. You just need to provide Web3Auth with a
+   JWT `id_token` and it will handle the key generation for you.
+3. You can create your wallet for any chain, depending on your needs. Web3Auth generates the private key which can be used for any chain.
+4. You can even use Web3Auth to create an HDKey and generate multiple sub accounts for your users. This is useful for wallets that support multiple
```

Comment:
What's HDKey?

Reply:
I'm referring to this: https://www.npmjs.com/package/hdkey

Reply:
Thanks for sharing. This was new info for me 😄